### DISCUSSION TITLE
LAN ports not working

### DISCUSSION THREAD

**Discussion by sjasbir** [4IceG/ZBT-Z8102AX-M2#5] [General]
I flashed the router with latest build from your repository. The WIFI networks come up and I am able to connect to GUI with it. However, none of the LAN ports work or have any DHCP IP provided. Do I need to configure anything for lan ports to work?

**Comment by 4IceG** [comment]
Hi, 
From what I remember, users reported such an error caused by modifying Wi-Fi settings. If I remember correctly, restarting the router fixed this situation.
